server/@types/express/index.d.ts

```diff
@@ -0,0 +1,9 @@
+export default {}
+
+declare module 'express-session' {
+  // Declare that the session will potentially contain these additional fields
+  interface SessionData {
+    returnTo: string
+    nowInMinutes: number
+  }
+}
```

Comment:
Really fiddly but added `./server/@types` as a definition root which allows overriding types in other packages.

Here we want to be able to say that the session might contain `returnTo` and `nowInMinutes`

We probably will also want to add the user object to res.locals but not currently required